### PR TITLE
chore: Remove redundant component

### PR DESCRIPTION
**Pull request by mukultayal135**
# Description

Converted to functional component RadioGroup and RadioGroupItem and linting
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist

- [ ] The title of the PR states what changed and the related issues number (used for the release note).
- [ ] Does this PR require documentation updates?
- [ ] I've updated documentation as required by this PR.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/Common/RadioGroup.tsx [hunk 1]
```diff
@@ -1,53 +1,22 @@
...
-                        onChange: this.props.onChange,
-                    }}
-                >
-                    {this.props.children}
-                </RadioGroupContext.Provider>
-            </div>
-        )
-    }
+const RadioGroup = (props: RadioGroupProps) => {
+    const { name, value, disabled, onChange, className, children } = props
```
Comment (vivek-devtron) [inline src/Common/RadioGroup.tsx:13]: We should directly extract it on line no 12

File: src/Common/RadioGroup/RadioGroup.tsx [hunk 2]
```diff
@@ -19,39 +20,53 @@ interface RadioGroupInterface {
     className?: string
     initialTab: string
     disabled: boolean
+    children: ReactNode
+}
+
+interface RadioInterface {
```
Comment (vivek-devtron) [inline src/Common/RadioGroup/RadioGroup.tsx:26]: Move interfaces in types file

File: src/Common/RadioGroupItem.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { RadioGroupItemProps } from './Types'
+import { RadioGroupContext } from './RadioGroup'
+
+const RadioGroupItem = (props: RadioGroupItemProps) => {
+    const { value, disabled, children, dataTestId } = props
```
Comment (vivek-devtron) [inline src/Common/RadioGroupItem.tsx:5]: Please extract the props on line no 4

File: src/Common/RadioGroup.tsx (hunk 1 above)
Comment (mukultayal135) [reply src/Common/RadioGroup.tsx:13]: done

File: src/Common/RadioGroupItem.tsx (hunk 3 above)
Comment (mukultayal135) [reply src/Common/RadioGroupItem.tsx:5]: done

File: src/Common/RadioGroup/RadioGroup.tsx (hunk 2 above)
Comment (mukultayal135) [reply src/Common/RadioGroup/RadioGroup.tsx]: done

**Review by vivek-devtron — DISMISSED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Review by AbhishekA1509 — APPROVED** [review]

**Pull request merged**